src/library/scala/collection/mutable/WrappedArray.scala

```diff
@@ -118,7 +118,7 @@ object WrappedArray {
         ArrayBuilder.make[T]()(m) mapResult WrappedArray.make[T]
   }
 
-  def newBuilder[A]: Builder[A, IndexedSeq[A]] = new ArrayBuffer
+  def newBuilder[T](implicit t: ClassTag[T]): WrappedArrayBuilder[T] = new WrappedArrayBuilder[T](t)
```

Comment:
The least would be for this function to return a `Builder[A, WrappedArrayBuilder[A]]`. `newBuilder` methods in the existing API always return a `Builder` in their public API, not a more specific type.